### PR TITLE
Update change babel-preset-es2015 to babel-preset-env

### PR DESCRIPTION
**Pull request by JPeer264**
**Please Read the [CONTRIBUTING Guidelines](https://github.com/babel/babel-loader/blob/master/CONTRIBUTING.md)**
*In particular the portion on Commit Message Formatting*

**Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)
`babel-preset-es2015` is used

**What is the new behavior?**
Changed to `babel-preset-env` in the readme and `.babelrc`

**Does this PR introduce a breaking change?**
- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the following...

* Impact:
* Migration path for existing applications:
* Github Issue(s) this is regarding:

**Other information**:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "babel-core": "^6.0.0",
     "babel-eslint": "^7.1.0",
     "babel-plugin-istanbul": "^3.0.0",
-    "babel-preset-es2015": "^6.0.0",
```
Comment (danez) [inline package.json]: babel-preset-es2015 is used in tests, so it needs to stay

File: .babelrc [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "presets": [
-    ["latest", { "es2015": { "loose": true } }]
```
Comment (danez) [inline .babelrc]: This is invalid, we use latest here no es2015

Comment (JPeer264) [inline .babelrc]: whoops, of course - thanks!

**Comment by JPeer264** [comment]
@danez should be valid now!

File: package.json [hunk 3]
```diff
@@ -22,7 +22,8 @@
...
+    "babel-preset-env": "^1.1.11",
```
Comment (hzoo) [inline package.json]: We don't need to add this useless we're going to use it on babel-loader itself

Comment (JPeer264) [inline package.json]: oh true, I forgot to remove that! Thanks.

**Comment by danez** [comment]
I'm okay with it as is, but if you want you could replace latest with env in package.json and .babelrc.
https://twitter.com/left_pad/status/837757756748759040
Otherwise I merge it.

**Pull request merged**